packages/dev/docs/src/docs.css

```diff
@@ -498,14 +498,17 @@ h2.sectionHeader {
 
 @define-mixin small-propTable {
   /* Collapse the prop table into a list with key/value pairs */
-  .propTable tr {
-    display: table;
-    width: 100%;
-    padding: 8px 0;
+  .propTable {
+    tr, td {
+      display: block;
+      text-align: left;
+    }
   }
 
-  .propTable td {
-    display: table-row;
+  .propTable tr td {
+    border: none !important;
```

Comment:
Why is the important necessary?

Reply:
it was easier and i forgot about it >.<
see new commit to see what was needed to get rid of it